### PR TITLE
Almost Latest Versions

### PR DESCRIPTION
**Pull request by thedevbob005**
Updated Vue, Axios, Vue-router, Vuex, Electron, Vue Loader and Style Loaders to Latest Possible Versions

### CODE REVIEW

File: template/package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
   {{#if_eq builder 'builder'}}
   "build": {
     "productName": "{{ name }}",
-    "appId": "org.simulatedgreg.electron-vue",
+    "appId": "com.example.{{ name }}",
```
Comment (XanderLuciano) [inline template/package.json]: May I suggest changing the appId line in the package.json template to include the author name in addition to the project name? 

    "appId": "com.{{ author }}.{{ name }}",

Comment (XanderLuciano) [reply template/package.json]: Oh wait, I just remembered you can include additional info like email in the author name... In which case you'd have to filter the input string.

**Comment by XanderLuciano** [comment]
You'll want to update the electron version property
Just change line 37:

    src/renderer/components/LandingPage/SystemInformation.vue

    // Currently (Will not work in 2.0.0 / 2.0.2)
    electron: process.versions['atom-shell'],

    // Should be updated to this property instead
    electron: process.versions.electron,

After that everything seems to work just fine!

image

**Comment by XanderLuciano** [comment]
@thedevbob005 Why did you close this PR?

**Comment by thedevbob005** [comment]
i can reopen this but that is after i fix my webpack 4

**Comment by thedevbob005** [comment]
I will sit again tomorrow and try to get it up and running properly. If it is a success I will make a new pull request.

**Pull request closed without merging**